sdks/python/apache_beam/io/hdfs_integration_test/py35/Dockerfile

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+# This image contains a Python SDK build and dependencies.
+# By default it runs wordcount against a locally accessible HDFS service.
+# See hdfs_integration_test.sh for example usage.
+
+FROM python:3.5
```

Comment:
Could we make base image a parameter here and pass it externally when we run docker to build the container image? This file could start like:
```
ARG BASE_IMAGE
FROM $BASE_IMAGE
```
however we'd need to figure out the gradle bits and the location of the dockerfile so that we could call it with multiple python versions.

Reply:
Also, I'd modify the python version in-place and double check that this test passes before doing major changes to the test/config structure. Perhaps you've already done that.

Reply:
Thanks for helping @tvalentyn , I made changes to the test structure to pass the python version as an argument, all hdfs tests are passing